src/Peddler/DoubleGenerator.cs

```diff
@@ -0,0 +1,237 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Peddler {
+    /// <summary>
+    ///   An implementation of <see cref="IDistinctGenerator{Double}" />.
+    /// </summary>
+    /// <inheritdoc />
+    public class DoubleGenerator : IComparableGenerator<Double> {
+        private static ThreadLocal<Random> random { get; } =
+            new ThreadLocal<Random>(() => new Random());
+
+        private const Int32 EpsilonExponent = -323;
+        private const Int32 MinimumIntervalExponentDelta = 16;
+
+        /// <summary>
+        /// The minimum value to generate in calls to <see cref="Next()" />. Defaults to
+        /// <see cref="Double.MinValue" />
+        /// </summary>
+        public Double Minimum { get; }
+
+        /// <summary>
+        /// The maximum value to generate in calls to <see cref="Next()" />. Defaults to
+        /// <see cref="Double.MaxValue" />.
+        /// </summary>
+        public Double Maximum { get; }
```

Comment:
The [`IIntegralGenerator`](https://github.com/invio/Peddler/blob/79e3a50e4c24be820c83bf03c0a0794e8dc8988f/src/Peddler/IIntegralGenerator.cs) implementations use "Low" and "High" rather than "Minimum" and "Maximum". Please update those to use Minimum and Maximum, or update these to be Low and High for the sake of consistency.